pkg/collect/host_collector.go

```diff
@@ -8,6 +8,8 @@ type HostCollector interface {
 	Title() string
 	IsExcluded() (bool, error)
 	Collect(progressChan chan<- interface{}) (map[string][]byte, error)
+	RemoteCollect(progressChan chan<- interface{}) (map[string][]byte, error) // RemoteCollect is used to priviledge pods to collect data from different nodes
+	HasRemoted() bool                                                         // HasRemoted returns true if the collector has a remote collector implementation
```

Comment:
I wonder if `HasRemoteCollector` would be more descriptive here. Just nitpicking mostly, will leave it up to you.

Reply:
Yes, this function is not easy to read. I have removed it. And use 
```
if err == collect.ErrRemoteCollectorNotImplemented {
```
to check

Reply:
updated to use
```
if errors.Is(err, collect.ErrRemoteCollectorNotImplemented) {
```